securedrop/tests/test_source.py

```diff
@@ -21,6 +24,9 @@
 from utils.instrument import InstrumentedApp
 
 overly_long_codename = 'a' * (Source.MAX_CODENAME_LEN + 1)
+SOURCE_ENDPOINTS = ['main.index', 'main.lookup', 'main.generate', 'main.login',
```

Comment:
We may want to add other endpoints called `allowed endpoints` for CSS and other static content and check that those endpoints aren't blocked by this feature.

Reply:
good point, done